src/query/remote/compressed_codecs.go

```diff
@@ -150,8 +150,11 @@ SeriesIterator also has a TagIterator representing the tags associated with it.
 
 This function transforms a SeriesIterator into a protobuf representation to be
 able to send it across the wire without needing to expand the series.
+
+If reset argument is true, the SeriesIterator readers will be reset so it can
+be iterated again. If false, the SeriesIterator will no longer be useable.
 */
-func compressedSeriesFromSeriesIterator(
+func CompressedSeriesFromSeriesIterator(
```

Comment:
This comment needs to start with `// CompressedSeriesFromSeriesIterator ...` now since it's an exposed type.

Also time to get rid of the `/*` and `*/` and use typical `//` for `godoc` to process it.

Reply:
Yeah this comment is rather verbose too, we can probably change it to be a general comment rather than as godoc on the function